wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -340,6 +341,11 @@ public WorkspaceStatus getStatus(String workspaceId) {
           createInternalEnvironment(
               environment, workspaceConfig.getAttributes(), workspaceConfig.getCommands());
       RuntimeContext runtimeContext = infrastructure.prepare(runtimeId, internalEnv);
+
+      internalEnv.getMachines()
```

Comment:
Please consider moving this logic to [InternalEnvironmentFactory](https://github.com/eclipse/che/blob/aaa8f424cd198e68e128bca939d9ac058b9ea52e/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/environment/InternalEnvironmentFactory.java)

Reply:
Thanks. Done.